index.html

```diff
@@ -618,6 +618,15 @@ <h2>Data Model and Representations</h2>
       <a href="https://www.w3.org/TR/did-core/#representations">Representations</a>.
     </p>
 
+    <p class="advisement">
+      There is no consensus regarding preserving data model properties accross representations.
+      Removing properties when converting a did document can result in an inability to maintain 
```

Comment:
There are terms in this PR, such as DID and DID Document, that need to be wrapped in \<a> tags so they're properly linked to the definitions.

Reply:
prefer to handle that in a separate editorial PR: https://github.com/w3c/did-imp-guide/issues/15